bin/winTM.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+. ./bin/import-shared
+
+log_usage () {
+	echo "Usage: $SCRIPTNAME {start|stop}" >&2
+	echo "We will start or stop all the configured TM Windows Services for you"
+	exit 3
+}
+
+do_start_stop() {
+  for i in $SERVICES
+  do 
+     ./bin/winSrvc.sh $1 $i  
+  done
+}
+
+case "$1" in
+  stop)
+  do_start_stop $1
+  ;;
+  start)
+  do_start_stop $1
+   ;;
+  *)
+	log_usage
+	;;
+esac
+
+:
```

Comment:
Really cool scripts

all we need now is some unit tests to confirm that all goes according to plan

see https://github.com/TeamMentor/TM_Docker/tree/tm-node/tests